test/components/Profile/ProfileImageField.test.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { ProfileImageField } from '../../../src/components/Profile';
+import '@testing-library/jest-dom/extend-expect';
+
+const MockProfileComponent = ({ noUserImage, mockClientProfile }) => {
+  if (!noUserImage) {
+    localStorage.setItem('profileImage', 'https://example.com/user.png');
+  }
+  return <ProfileImageField clientProfile={mockClientProfile} />;
+};
+
+describe('ProfileImageField', () => {
+  it('renders Choose Img button and PersonIcon if clientProfile is null and user has no profile img', () => {
+    const { queryByRole, queryByTestId } = render(
+      <MockProfileComponent noUserImage mockClientProfile={null} />
+    );
+    const buttonElement = queryByRole('button');
+    expect(buttonElement.textContent).toBe('Choose Img');
+
+    const svgElement = queryByTestId('PersonIcon');
+    expect(svgElement).not.toBeNull();
+  });
+
+  it('renders Remove Img button and image if clientProfile is null, but user has profile image', () => {
+    const { queryByRole, queryByTestId } = render(
+      <MockProfileComponent mockClientProfile={null} />
+    );
+    const buttonElement = queryByRole('button');
+    expect(buttonElement.textContent).toBe('Remove Img');
+
+    const svgElement = queryByTestId('PersonIcon');
+    expect(svgElement).toBeNull();
+  });
+
+  it('renders no button with PersonIcon if clientProfile is not null and has no profile image', () => {
+    const { queryByRole, queryByTestId } = render(<MockProfileComponent mockClientProfile={{}} />);
+    const buttonElement = queryByRole('button');
+    expect(buttonElement).toBeNull();
+
+    const svgElement = queryByTestId('PersonIcon');
+    expect(svgElement).not.toBeNull();
+  });
+
+  it('renders no button with image with clientprofile is not null and has profile image', () => {
```

Comment:
small typo:
`with clientprofile` should be `if clientProfile`

Reply:
Got this fixed.